scipy/optimize/__minpack.h

```diff
@@ -225,7 +331,7 @@ int jac_multipack_lm_function(int *m, int *n, double *x, double *fvec, double *f
       *iflag = -1;
       return -1;
     }
-    if (multipack_jac_transpose == 1) 
+    if (multipack_jac_transpose == 1)
```

Comment:
Multipack? This code looks ancient. Amazing that this C wrapper worked for 2 decades still.

Reply:
Yes, mostly because NumPy C API is still working the same. That's why it is working. Basically all these macros are nannying the python objects for f2py generated C code. From this I learned a bit about the mechanism and now I have an idea for bypassing Cython and providing directly a C extension module.